server/swagger_v2.yml

```diff
@@ -0,0 +1,482 @@
+openapi: "3.0.0"
+info:
+  title: Gateway Service
+  version: 0.1.0
+servers:
+  - url: /chronograf/v2
+paths:
+  /cells:
+    post:
+      tags:
+        - Cells
+      summary: Create a cell
+      requestBody:
+          description: cell to create
+          required: true
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Cell"
+      responses:
+        '201':
+          description: Added cell
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Cell"
+        default:
+          description: unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+    get:
+      tags:
+        - Cells
+      summary: Get all cells
+      parameters:
+        - in: query
+          name: organization
+          schema:
+            type: string
+          description: filter for dashboards belonging to a particular organization
+        - in: query
+          name: organizationID
+          schema:
+            type: string
+          description: filter for dashboards belonging to a particular organizationID
+        - in: query
+          name: user
+          schema:
+            type: string
+          description: filter for dashboards belonging to a particular user
+        - in: query
+          name: userID
+          schema:
+            type: string
+          description: filter for dashboards belonging to a particular userID
+      responses:
+        '200':
+          description: all cells
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Cells"
+        default:
+          description: unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+  '/cells/{cellID}':
+   get:
+    tags:
+      - Cells
+    summary: Get a single Cell
+    parameters:
+        - in: path
+          name: cellID
+          schema:
+            type: string
+          required: true
+          description: ID of cell to update
+    responses:
+        '200':
+          description: get a single cell
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Cell"
+        default:
+          description: unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+   patch:
+      tags:
+        - Cells
+      summary: Update a single cell
+      requestBody:
+          description: patching of a cell
+          required: true
```

Comment:
Ok, I agree with that the body should be required.  So the test above that didn't check for empty body should error.